### PR TITLE
Anchor buff / balance units with anchor

### PR DESCRIPTION
**Pull request by Ithilis**
This comit is linked with all anchored units that also contain balance
changes while its linked together.

T1 land scout -> remove artilery support while it was ussles on unit
that die in 0second.
-- increse range that was stupidly low, and adjust radar

T2 mml Avalanche -> replace anchor with bombard mode,

T2 tank Brute -> move into 300m category, add anchor. its obsidian style
unit in lower class. change projectile to use gravity.

T2 field enginer Scarab -> decrese range of tmd to fit with buffed
anchor, change build icon proority to be near enginer, increse e
maintaince.

T3 mAA -> decrese range to better fit with buffed anchor

T3 tank Slugger -> add anchor, and balance (more hp, less speed)

### CODE REVIEW

**Comment by Ithilis** [comment]
yes this is damn huge pr. but i dont know how to do it while this is all linked into buffed anchor ability. 

WARNING!!!!
-> there is mistake that i remove SILO in mml. In next comit it should take it back.

File: units/INU2003/INU2003_unit.bp [hunk 1]
```diff
@@ -62,7 +61,6 @@ UnitBlueprint {
         'LAND',
         'TECH2',
         'INDIRECTFIRE',
-        'SILO',
```
Comment (Ithilis) [inline units/INU2003/INU2003_unit.bp]: i remove it by mistake :/ and now its too much work to make this stupidly huge comit again. After merge make new comit that get it back.

File: loc/US/nomads_strings_core.lua [hunk 2]
```diff
@@ -68,15 +68,15 @@ tooltip_lobby_unit_restriction_nonomads_desc = ""
 tooltip_radar_boost_title = "Radar Boost Toggle"
 tooltip_radar_boost_desc = "Starts/stops charging radar boosting for the selection units. When finished charging up radar boost initiates automatically. Charging up requires energy."
 tooltip_bombardmode_title = "Bombard Mode Toggle"
-tooltip_bombardmode_desc = "Turn the selection units bombard mode on/off"
+tooltip_bombardmode_desc = "bombard mode decrese reload time by 50% but increse firing randomnes"
```
Comment (CookieNoob) [inline loc/US/nomads_strings_core.lua]: plz fix spelling errors:
"Bombardment mode decreases the reload time by 50% but increases the firing randomnes."

File: loc/US/nomads_strings_core.lua [hunk 3]
```diff
@@ -68,15 +68,15 @@ tooltip_lobby_unit_restriction_nonomads_desc = ""
...
 tooltip_no_air_title = "Anti-Air Toggle"
 tooltip_no_air_desc ="Turn the selection units anti-air attack on/off"
 tooltip_snipermode_title = "Sniper Mode Toggle"
 tooltip_snipermode_desc = "Turn sniper mode on/off"
 tooltip_usecapacitor_title = "Capacitor Toggle"
 tooltip_usecapacitor_desc = "Boost unit temporarily (firepower, build power, regeneration) using the capacitor ability"
 tooltip_anchor_title = "Anchor Toggle"
-tooltip_anchor_desc = "Immobilizes the unit"
+tooltip_anchor_desc = "Immobilizes the unit and increse weapon range, vision, radar and build rate by 30%"
```
Comment (CookieNoob) [inline loc/US/nomads_strings_core.lua]: fix spelling:
"Immobilizes the unit and increases weapon range, vision, radar and build rate by 30%."

File: loc/US/nomads_strings_core.lua [hunk 4]
```diff
@@ -686,14 +686,14 @@ inu3007_desc = "Mobile Anti-Air Missile Launcher"
 inu3007_help = "Launches anti-air missiles at enemy aircraft."
 inu3007_name = "Watchman"
 
--- T3 mobile missile defense
+-- T2 mobile missile defense
 inu3008_desc = "Field Engineer"
-inu3008_help = "Hover unit equipped with limited engineering capabilities and a tactical missile defense installation. Can only construct offensive and defensive structures."
+inu3008_help = "Hover unit equipped with engineering capabilities and a tactical missile defense installation. Have anchor option for higer intel and build capacity."
```
Comment (CookieNoob) [inline loc/US/nomads_strings_core.lua]: Spelling:
Has anchor option to increase intel and tmd range as well as the build capacity."

File: loc/US/nomads_strings_core.lua [hunk 5]
```diff
@@ -686,14 +686,14 @@ inu3007_desc = "Mobile Anti-Air Missile Launcher"
...
 inu3008_name = "Scarab"
 
 -- T3 tracked tank
 inu3009_desc = "Armored Assault Tank"
-inu3009_help = "Tracked tank unit that carries dual plasma cannons and very heavy armor."
+inu3009_help = "Tank that carries dual cannons and heavy armor. Have anchor option for defensive use."
```
Comment (CookieNoob) [inline loc/US/nomads_strings_core.lua]: Have -> Has

File: projectiles/NParticleBlast1/NParticleBlast1_proj.bp [hunk 6]
```diff
@@ -36,6 +36,6 @@ ProjectileBlueprint {
         DestroyOnWater = false,
         InitialSpeed = 35,
         VelocityAlign = true,
-        UseGravity = false, -- should make unit not get stuck shooting AA
+        UseGravity = true, -- should make unit not get stuck shooting AA
```
Comment (CookieNoob) [inline projectiles/NParticleBlast1/NParticleBlast1_proj.bp]: what does this do?

File: units/INU1008/INU1008_unit.bp [hunk 7]
```diff
@@ -1,4 +1,4 @@
--- tank destroyer
+-- Tank destroyer
```
Comment (CookieNoob) [inline units/INU1008/INU1008_unit.bp]: this is part of your bp-name-update PR and will lead to conficts

File: units/INU2003/INU2003_Script.lua [hunk 8]
```diff
@@ -1,17 +1,18 @@
 -- T2 missile launcher
 
+local AddBombardModeToUnit = import('/lua/nomadsutils.lua').AddBombardModeToUnit
```
Comment (CookieNoob) [inline units/INU2003/INU2003_Script.lua]: bombardment mode is not anchor, why is this part of this PR?

File: units/INU2003/INU2003_unit.bp [hunk 9]
```diff
@@ -1,10 +1,9 @@
---T2 mml Avalanche
+-- T2 mml Avalanche
```
Comment (CookieNoob) [inline units/INU2003/INU2003_unit.bp]: part of the bp-update-PR

File: units/INU2003/INU2003_unit.bp (hunk 1 above)
Comment (CookieNoob) [inline units/INU2003/INU2003_unit.bp]: then just add it back by pushing a commit into this branch which adds the line back

File: projectiles/NParticleBlast1/NParticleBlast1_proj.bp (hunk 6 above)
Comment (Ithilis) [inline projectiles/NParticleBlast1/NParticleBlast1_proj.bp]: It get very small angle to projectile, so insted of stred line its more like uef projectiles (but smaller angle)

File: units/INU2003/INU2003_Script.lua (hunk 8 above)
Comment (Ithilis) [inline units/INU2003/INU2003_Script.lua]: becaue anchor go down and is replaced by bombardment. So when it only remove anchor then will need make another comit that also add bombardment.  And one comit with other will not have balanced unit.

**Pull request merged**